src/huggingface_hub/templates/modelcard_template.md

```diff
@@ -32,6 +32,12 @@
 - **Paper [optional]:** {{ paper | default("[More Information Needed]", true)}}
 - **Demo [optional]:** {{ demo | default("[More Information Needed]", true)}}
 
+## How to Get Started with the Model
+
+Use the code below to get started with the model.
+
+{{ get_started_code | default("[More Information Needed]", true)}}
+
```

Comment:
To keep the Uses, Bias, Risks, and Limitations as a focus, I could see the *How to Get Started* section being before *Training Details* (after *Recommendations*)